staging/kube-prometheus-stack/patch/patch_9_mesosphere_helmignore.sh

```diff
@@ -15,6 +15,8 @@ cat << EOF >> ${SRCFILE}
 upgrade_operator.sh
 kube-prometheus-stack-*.tgz
 patch/
+CONTRIBUTING.md
+UPGRADING.md
```

Comment:
same - I don't see `UPGRADING.md` in this branch or in master - does that matter?